example.ini

```diff
@@ -43,6 +43,11 @@ base-url =
 ; secret token for communication with harold
 secret =
 
+[elasticsearch]
```

Comment:
That comment explains why we have elasticsearch, but not what should go into the parameters. What is index? what is type?